apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -645,7 +646,12 @@ export default function Success(props: PageProps) {
 
                           return (
                             <>
-                              <div className="text-emphasis mt-4 font-medium">{label}</div>
+                              <div
+                                className="text-emphasis mt-4 font-medium"
+                                dangerouslySetInnerHTML={{
+                                  __html: markdownToSafeHTML(label),
```

Comment:
`markdownToSafeHTML` has huge bundle size.
![image](https://github.com/user-attachments/assets/a21bc9a1-1899-4cdb-b480-80be3d5c2696)

Can we run it server side only

Reply:
Yeah, this make sense👍